shared/src/test/scala/amf/core/render/GovernanceModeTest.scala

```diff
@@ -0,0 +1,72 @@
+package amf.core.render
+
+import amf.core.client.scala.AMFGraphConfiguration
+import amf.core.client.scala.config.RenderOptions
+import amf.core.client.scala.model.document.Document
+import amf.core.client.scala.model.domain.{ObjectNode, SerializableAnnotation}
+import amf.core.internal.annotations.{DeclaredElement, TrackedElement}
+import amf.core.internal.parser.domain.Annotations
+import amf.core.internal.remote.Spec.AMF
+import amf.core.io.FileAssertionTest
+import org.scalatest.funsuite.AsyncFunSuite
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class GovernanceModeTest
```

Comment:
@hghianni can you add 3 more tests. Each test should parse a RAML, OAS & ASYNC API respectively, resolve with editing resolution and compare the dumped JSON-LD with a golden. This is just to replicate the flow currently in APID & Governance